xbmc/settings/AdvancedSettings.cpp

```diff
@@ -764,6 +764,9 @@ void CAdvancedSettings::ParseSettingsFile(const std::string &file)
         if (XMLUtils::GetFloat(pRefreshVideoLatency, "delay", delay, -600.0f, 600.0f))
           videolatency.delay = delay;
 
+        if (XMLUtils::GetFloat(pRefreshVideoLatency, "hdrlatency", hdrlatency, -600.0f, 600.0f))
```

Comment:
For consistency, hdrdelay would be better I think. or hdrdelta if it remains a delta value.

Reply:
I'll rename it "hdrdelta" for now. Will see if it becomes an absolute value, it'll be hdrdelay indeed. 
Thank you

Reply:
that name is a bit weird though, maybe hdrextradelay is easier to understand without documentation?

Reply:
'hdrextradelay' actually my 1st choice, then scrapped it... Sold. Thanks.